dev-util/pycharm-community/pycharm-community-2024.3.2.ebuild

```diff
@@ -0,0 +1,202 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop readme.gentoo-r1 wrapper xdg
+
+DESCRIPTION="Intelligent Python IDE with unique code assistance and analysis"
+
+# NOTE upstream release info
+# https://data.services.jetbrains.com/products?code=PCP,PCC&release.type=release
+# https://data.services.jetbrains.com/products?code=PCP,PCC&release.type=release&fields=name,releases
+
+HOMEPAGE="https://www.jetbrains.com/pycharm/"
+SRC_URI="
+	amd64? (
+		https://download.jetbrains.com/python/${P}.tar.gz
+	)
+	arm64? (
+		https://download.jetbrains.com/python/${P}-aarch64.tar.gz
+	)
+	x86? (
+		https://download.jetbrains.com/python/${P}.tar.gz
+	)
+"
+
+if [[ ${PN} == *-professional ]]; then
+	S="${WORKDIR}/${PN/%-professional/}-${PV}"
+	LICENSE="|| ( JetBrains-business JetBrains-classroom JetBrains-educational JetBrains-individual )
+	Apache-2.0 BSD BSD-2 CC0-1.0 CC-BY-2.5 CC-BY-3.0 CC-BY-4.0 CPL-1.0 CDDL CDDL-1.1 EPL-1.0 EPL-2.0
+	GPL-2 GPL-2-with-classpath-exception ISC JDOM LGPL-2.1 LGPL-3 MIT MPL-1.1 MPL-2.0 OFL-1.1
+	PYTHON Unicode-DFS-2016 Unlicense UPL-1.0 ZLIB"
+else
+	LICENSE="|| ( JetBrains-business JetBrains-classroom JetBrains-educational JetBrains-individual )
+	Apache-2.0 BSD BSD-2 CC0-1.0 CC-BY-2.5 CC-BY-3.0 CPL-1.0 CDDL-1.1 EPL-1.0 GPL-2
+	GPL-2-with-classpath-exception ISC JDOM JSON LGPL-2+ LGPL-2.1 LGPL-3 MIT MPL-1.1 MPL-2.0
+	OFL-1.1 UPL-1.0 ZLIB"
+fi
+
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm64 ~x86"
+IUSE="+bundled-jdk"
+
+if [[ ${PN} == *-professional ]]; then
+	IUSE+=" +bundled-xvfb"
+fi
+
+BDEPEND="dev-util/debugedit
+	dev-util/patchelf
+"
+
+# NOTE
+#  The remote-dev-server present in pycharm-professional contains most of the
+#  libraries need for the bundled jdk. These are not in the RUNPATH of the jdk.
+#  So the dependencies are actually needed.
+RDEPEND="
+	!bundled-jdk? (
+		>=virtual/jre-1.8
+	)
+	bundled-jdk? (
+		app-accessibility/at-spi2-core:2
+		dev-libs/expat
+		dev-libs/glib:2
+		dev-libs/nspr
+		dev-libs/nss
+		dev-libs/wayland
+		media-libs/alsa-lib
+		media-libs/freetype
+		media-libs/mesa
+		net-print/cups
+		sys-apps/dbus
+		sys-libs/zlib
+		x11-libs/cairo
+		x11-libs/libX11
+		x11-libs/libXcomposite
+		x11-libs/libXcursor
+		x11-libs/libXdamage
+		x11-libs/libXext
+		x11-libs/libXfixes
+		x11-libs/libXi
+		x11-libs/libXrandr
+		x11-libs/libXrender
+		x11-libs/libXtst
+		x11-libs/libXxf86vm
+		x11-libs/libdrm
+		x11-libs/libxcb
+		x11-libs/libxkbcommon
+		x11-libs/pango
+	)
+"
+
+if [[ ${PN} == *-professional ]]; then
+RDEPEND+="
+	bundled-xvfb? (
```

Comment:
\+ `sys-libs/pam`, according to `qa-vdb`.

Reply:
> + `sys-libs/pam`, according to `qa-vdb`.

Ye, checked as well before, but forgot to add it. Apologies